Adding author profile for New Student

naukilina opened the pull request.

naukilina committed:
Adding author profile for New Student

naukilina committed:
Creating Review Article of Living Archives by Montse

naukilina committed:
Update 2018-18-10-review-living-archive.md

naukilina committed:
Update 2018-18-10-review-living-archive.md

naukilina committed:
Update 2018-18-10-review-living-archive.md

halperta committed:
Rename 2018-18-10-review-living-archive.md to 2018-10-18-review-living-archive.md

halperta commented on _posts/blog/2018-10-18-review-living-archive.md:
```diff
@@ -0,0 +1,20 @@
+Review of the Living Archive of Aboriginal Languages
```
Please add a YAML header, following the instructions in the tutorial:

http://www.halperta.com/criticalarchives/tutorial/writing-in-markdown/

halperta commented on _posts/blog/2018-10-18-review-living-archive.md:
```diff
@@ -0,0 +1,20 @@
+
+Living Archive of Aboriginal Languages. Charles Darwin University et al, 2012, http://laal.cdu.edu.au/. Accessed 17 oct. 2018
+
+Simplicity is one of the key words for the Living Archive of Aboriginal Languages. This digital repository contains over 2500 books written in more than 25 indigenous languages of the Northern Territory of Australia, but the texts are available through a very easy-to-use interface displaying a clickable geographical map, with icons that redirect the user to the materials. Another key word for thi ...
```
I might separate this into two  or even three paragraphs: one that begins "Another key word..." and a second that begins "Why were these texts..." That will help support the conversational tone of the writing.

halperta replied:
I don't think "but" is the correct way to connect the two ideas in the second sentence. Consider starting a new sentence instead.

halperta commented on _posts/blog/2018-10-18-review-living-archive.md:
```diff
@@ -0,0 +1,20 @@
+
+Simplicity is one of the key words for the Living Archive of Aboriginal Languages. This digital repository contains over 2500 books written in more than 25 indigenous languages of the Northern Territory of Australia, but the texts are available through a very easy-to-use interface displaying a clickable geographical map, with icons that redirect the user to the materials. Another key word for thi ...
+
+The Living Archive of Aboriginal Languages was created in 2012 by the Charles Darwin University, the Australian National University and the NT Department of Education and Children’s Services, with funding from the Australian Research Council through the Linkage Infrastructure Equipment and Facilities scheme. On a second phase, the archive received financial aid again from the Research Council, an ...
```
Sentence 2: "For a second phase..."
Final sentence: "Which from 1973..."

halperta commented on _posts/blog/2018-10-18-review-living-archive.md:
```diff
@@ -0,0 +1,20 @@
+
+The Living Archive of Aboriginal Languages was created in 2012 by the Charles Darwin University, the Australian National University and the NT Department of Education and Children’s Services, with funding from the Australian Research Council through the Linkage Infrastructure Equipment and Facilities scheme. On a second phase, the archive received financial aid again from the Research Council, an ...
+
+The “centres” were part of a discontinued Bilingual Program of northern Australia that aimed for the literacy of aboriginal children. Indigenous and non-indigenous people worked together to write and illustrate the books that contain “teaching materials, literacy primers, children’s stories, stories of local cultural significance such as environmental knowledge, traditional practices, oral litera ...
```
Sentence 1: "aimed to improve the literacy..."?

halperta commented on _posts/blog/2018-10-18-review-living-archive.md:
```diff
@@ -0,0 +1,20 @@
+
+The “centres” were part of a discontinued Bilingual Program of northern Australia that aimed for the literacy of aboriginal children. Indigenous and non-indigenous people worked together to write and illustrate the books that contain “teaching materials, literacy primers, children’s stories, stories of local cultural significance such as environmental knowledge, traditional practices, oral litera ...
+
+Access is an guiding force for this archive, but for whom is the key question. Jeannette A. Bastian states “the creation of collective memory by nations, communities, or groups of people depends upon their ability to access their own cultural heritage” (81). For Bastian, access should be a fundamental component of custody. The Living Archive takes this insightful argument in practice as an exampl ...
```
Sentence 1: "is **a** guiding force"
Sentence 5 (the project does not include...): this is a bit confusing. Maybe just "the project does not keep.... instead, it is invested"?
Final sentence: this is also a bit confusing, perhaps because it's hard to identify the subject of the verb "depending". Is it the cultural heritage, or the communities? It might be simpler as two sentences.

halperta commented on _posts/blog/2018-10-18-review-living-archive.md:
```diff
@@ -0,0 +1,20 @@
+
+Access is an guiding force for this archive, but for whom is the key question. Jeannette A. Bastian states “the creation of collective memory by nations, communities, or groups of people depends upon their ability to access their own cultural heritage” (81). For Bastian, access should be a fundamental component of custody. The Living Archive takes this insightful argument in practice as an exampl ...
+
+There is evidence of an accountability and ethical responsibility to the indigenous peoples of the Northern Territory from the first time one enters the website. A notice that reads “Respecting ownership” is at its gate. It indicates that the contents belong to the “Aboriginal language owners, creators of the materials and their descendants.” The notice also serves as a warning to the Aboriginal  ...
```
Corrections:
* s1: this phrasing is ambiguous. There is evidence that the project is holding itself accountable? That they are fulfilling an ethical responsibility?
* available for *download*
* once *entering* the website
* *Rights* statement
* "permission to publish, *and of calling on* users to reach out if..."

halperta commented on _posts/blog/2018-10-18-review-living-archive.md:
```diff
@@ -0,0 +1,20 @@
+
+There is evidence of an accountability and ethical responsibility to the indigenous peoples of the Northern Territory from the first time one enters the website. A notice that reads “Respecting ownership” is at its gate. It indicates that the contents belong to the “Aboriginal language owners, creators of the materials and their descendants.” The notice also serves as a warning to the Aboriginal  ...
+
+To sum up, or more accurately to start the conversation some conclusions are in hand. It is important to acknowledge that the Living Archive is not community-based (following Michelle Caswell’s reflections), which would be a step forward in the decolonization of indigenous knowledges. It is not the case because the archive was designed, and today is managed mostly by powerful knowledge-making uni ...
```
Corrections:
* S1: comma after conversation
* S1: "conclusions are *called for*"
* creators, *and scanning* the materials...
* special *attitude*
* "sensible significance" is redundant. Just say "significance"

halperta replied:
Again, you might break this into two paragraphs, starting with "what is at stake..."

halperta commented on _posts/blog/2018-10-18-review-living-archive.md:
```diff
@@ -0,0 +1,20 @@
+Works Cited
+
+Bow, Catherine et al. “Developing a Living Archives of Aboriginal Languages”. Language Documentation and Conservation. Vol. 8, 2014, pp. 345-360.
+Bastian, Jeannette. “Taking Custody, Giving Access: A Postcustodial role for a new Century”.   
```
Maybe add the Michelle Caswell citation?

halperta reviewed (changes requested):
I'm not familiar with this project, and the collection sounds fascinating and your reflections on its potential as a decolonial project are really promising. You don't have to address these questions in this piece, but I find myself wondering: what do you mean by "decolonial" here? I generally define decolonial as undoing colonial structures; can a digital database run by a university accomplish that?

I've made some editorial comments below: identified grammatical concerns or typos, and made a few recommendations about paragraph breaks. Please make your revisions by next Monday, oct 29. Feel free to write with any questions.

naukilina committed:
Update 2018-10-18-review-living-archive.md

naukilina committed:
Update 2018-10-18-review-living-archive.md

naukilina committed:
Update 2018-10-18-review-living-archive.md

naukilina committed:
Update 2018-10-18-review-living-archive.md

naukilina committed:
Update 2018-10-18-review-living-archive.md

naukilina committed:
Update 2018-10-18-review-living-archive.md

naukilina committed:
Update 2018-10-18-review-living-archive.md

naukilina committed:
Update 2018-10-18-review-living-archive.md

naukilina committed:
Update 2018-10-18-review-living-archive.md

naukilina committed:
Update 2018-10-18-review-living-archive.md

naukilina committed:
Update 2018-10-18-review-living-archive.md

naukilina committed:
Update 2018-10-18-review-living-archive.md

naukilina committed:
Update 2018-10-18-review-living-archive.md

naukilina committed:
Update 2018-10-18-review-living-archive.md

naukilina committed:
Update 2018-10-18-review-living-archive.md

naukilina committed:
Update 2018-10-18-review-living-archive.md

naukilina committed:
Update 2018-10-18-review-living-archive.md

naukilina committed:
Update 2018-10-18-review-living-archive.md

naukilina committed:
Update 2018-10-18-review-living-archive.md

naukilina committed:
Update 2018-10-18-review-living-archive.md

naukilina committed:
Update 2018-10-18-review-living-archive.md

naukilina committed:
Update 2018-10-18-review-living-archive.md

halperta committed:
Merge branch 'gh-pages' into gh-pages

halperta commented on _posts/blog/2018-10-18-review-living-archive.md:
```diff
@@ -0,0 +1,47 @@
+---
+layout: post
+title: "The Living Archive of Aboriginal Languages: A big step forwards giving back"
```
Do you mean "a big step towards giving back"?

halperta committed:
correcting typos

halperta reviewed (changes requested):
Montse - this is so close to being complete! I corrected a couple of typos, and just have one question left about the title: did you mean to write "towards" instead of "forwards"? As soon as you respond to that, I'll go ahead and publish!

naukilina commented:
Towards! Yes! Thank you mil


Sent via the Samsung Galaxy Express 3, an AT&T 4G LTE smartphone
-------- Mensaje original --------De: Hannah Alpert-Abrams <notifications@github.com> Fecha: 1/11/18  8:48 PM  (GMT-06:00) A: halperta/criticalarchives <criticalarchives@noreply.github.com> CC: naukilina <montserratmadariaga@utexas.edu>, Author <author@noreply.github.com> Asunto: Re: [halperta/criticalarchives] Adding author profile for New Student
  (#17) 
@halperta requested changes on this pull request.

Montse - this is so close to being complete! I corrected a couple of typos, and just have one question left about the title: did you mean to write "towards" instead of "forwards"? As soon as you respond to that, I'll go ahead and publish!

In _posts/blog/2018-10-18-review-living-archive.md:
> @@ -0,0 +1,47 @@
+---
+layout: post
+title: "The Living Archive of Aboriginal Languages: A big step forwards giving back"

Do you mean "a big step towards giving back"?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or mute the thread.
{"api_version":"1.0","publisher":{"api_key":"05dde50f1d1a384dd78767c55493e4bb","name":"GitHub"},"entity":{"external_key":"github/halperta/criticalarchives","title":"halperta/criticalarchives","subtitle":"GitHub repository","main_image_url":"https://assets-cdn.github.com/images/email/message_cards/header.png","avatar_image_url":"https://assets-cdn.github.com/images/email/message_cards/avatar.png","action":{"name":"Open in GitHub","url":"https://github.com/halperta/criticalarchives"}},"updates":{"snippets":[{"icon":"PERSON","message":"@halperta requested changes on #17"}],"action":{"name":"View Pull Request","url":"https://github.com/halperta/criticalarchives/pull/17#pullrequestreview-170944431"}}}
[
{
"@context": "http://schema.org",
"@type": "EmailMessage",
"potentialAction": {
"@type": "ViewAction",
"target": "https://github.com/halperta/criticalarchives/pull/17#pullrequestreview-170944431",
"url": "https://github.com/halperta/criticalarchives/pull/17#pullrequestreview-170944431",
"name": "View Pull Request"
},
"description": "View this Pull Request on GitHub",
"publisher": {
"@type": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
},
{
"@type": "MessageCard",
"@context": "http://schema.org/extensions",
"hideOriginalBody": "false",
"originator": "AF6C5A86-E920-430C-9C59-A73278B5EFEB",
"title": "@halperta requested changes on 17",
"sections": [
{
"text": "Montse - this is so close to being complete! I corrected a couple of typos, and just have one question left about the title: did you mean to write \"towards\" instead of \"forwards\"? As soon as you respond to that, I'll go ahead and publish!",
"activityTitle": "**Hannah Alpert-Abrams**",
"activityImage": "https://assets-cdn.github.com/images/email/message_cards/avatar.png",
"activitySubtitle": "@halperta",
"facts": [

]
}
],
"potentialAction": [
{
"targets": [
{
"os": "default",
"uri": "https://github.com/halperta/criticalarchives/pull/17#pullrequestreview-170944431"
}
],
"@type": "OpenUri",
"name": "View on GitHub"
},
{
"name": "Unsubscribe",
"@type": "HttpPOST",
"target": "https://api.github.com",
"body": "{\n\"commandName\": \"MuteNotification\",\n\"threadId\": 397752083\n}"
}
],
"themeColor": "26292E"
}
]

halperta committed:
typo
